### ISSUE TITLE
versioning app.json

### ISSUE THREAD

**Issue by pasindud** [AliasIO/Wappalyzer#800]
Can key called change count / version field be added or is easies to track whether there have been any changes

**Comment by AliasIO** [comment]
I don't follow, can you rephrase?

**Comment by pasindud** [comment]
I mean something like 

```
{
        "change_count":1   
	"apps": {
```

And keep the change count increasing so if someone else is also using this app.json will know how many changes he is behind. Can be something other than change count.

**Comment by AliasIO** [comment]
That's what version control is for, why wouldn't you use Git for this?

**Comment by pasindud** [comment]
would be easier in cases where you want only to update app.json and nothing else, and just copy past the app.json, without having git clone and having to do a diff

**Comment by AliasIO** [comment]
I don't think this scenario is common enough to manually maintain a version number. Sometimes people work on different changes at the same time. Git is perfect for this.
